packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -90,9 +94,10 @@ const ProgressBar = ({
         aria-valuemax={100}
         aria-valuemin={0}
         aria-valuenow={normalizedValue}
-        aria-valuetext={!isMac() ? `${normalizedValue}%` : undefined}
+        aria-valuetext={valText}
         tabIndex="-1"
       />
+      <VisuallyHiddenText aria-live="polite" text={valText} />
```

Comment:
```suggestion
      {valueText  && <VisuallyHiddenText aria-live="polite" text={valText} />}
```
Let's add visual hidden text only when `valueText` prop is specified

Reply:
Updated.
19176f6285654364b9774ee21c5e682ecead1a5f
Thanks